utils/qr_pay_generator.py

```diff
@@ -0,0 +1,69 @@
+import os
+from io import BytesIO
+
+import qrcode
+from django.conf import settings
+from django.core.files.base import ContentFile
+
+from camp.models import BankAccount, SummerCampInfo
+
+
+class QRPayGenerator:
```

Comment:
I think we should use a better name, like `QRCodeGenerator`, because this class does not generate QR pay, but QR code.